gcc_7-centos6/Dockerfile

```diff
@@ -22,53 +22,34 @@ RUN yum update -y \
        glibc-devel \
        glibc-devel.i686 \
        libgcc.i686 \
-       gmp-devel \
-       mpfr-devel \
-       libmpc-devel \
        nasm \
-       m4 \
        libffi-devel \
        openssl-devel \
        pkgconfig \
        subversion \
        zlib-devel \
        xz \
        curl \
-       xz-devel \
```

Comment:
why did you exclude xz ?

Reply:
Trying to drop `-devel` packages if they weren't needed to build the image, but I don't think there is any harm in leaving it.

Reply:
> @dheater please, replicate your changes to gcc_7-centos6-x86

Done. Tested by building `libcurl` `gettext_installer` and `gettext` on both `x86` and `64` versions.